src/index.html

```diff
@@ -1,17 +1,170 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field_style">
+        <legend>Personal information</legend>
+        <div class="input_style">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="input_style">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="input_style">
+          <label for="old">How old are You?</label>
+          <input
+            type="number"
+            id="old"
+            name="old"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+        <div class="input_style">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            id="date"
+            name="date"
+          >
+        </div>
+        <div>
+          <label for="terms">I accept the term of the agreement</label>
+          <input type="checkbox"
+            id="terms"
+            name="terms"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="field_style">
+        <legend>Registration</legend>
+        <div class="input_style">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div>
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="field_style">
+        <legend>An interesting fact about you!</legend>
+        <div class="input_style"> Do you love cats?
+          <input
```

Comment:
```suggestion
        <div class="input_style">
          Do you love cats?
          <input
```
text is also the child of this `div`, so better to write it in the new line, the same as other children

Reply:
FYI